statsmodels/tsa/stattools.py

```diff
@@ -1295,6 +1295,149 @@ def levinson_durbin_pacf(pacf, nlags=None):
     return arcoefs, acf
 
 
+def breakvar_heteroskedasticity_test(resid, subset_length=1/3,
+                                     alternative='two-sided', use_f=True):
+    r"""
+    Test for heteroskedasticity of residuals
+
+    Tests whether the sum-of-squares in the first subset of the sample is
+    significantly different than the sum-of-squares in the last subset
+    of the sample. Analogous to a Goldfeld-Quandt test. The null hypothesis
+    is of no heteroskedasticity.
+
+    Parameters
+    ----------
+    resid : array_like
+        Residuals of a time series model.
```

Comment:
Can you add a note about the expected shape of `resid`?  (see below for a suggestion to use `(nobs, nvars)` rather than `(nvars, nobs)`)

Reply:
I found it would be better to allow only 1-d input for this function and iterate over the `model.k_endog` in `test_heteroskedasticity` method as `test_normality` and `test_serial_correlation` do.

Reply:
Then `(nobs,)` would be the correct description.

Reply:
Now, 2-d input is allowed by small modification.
I will add a note as `(nobs,) or (nobs, nvars)`

Reply:
> I will add a note as (nobs,) or (nobs, nvars)

This looks like it still needs to be done.